etc/nginx/sites-available/wazo

```diff
@@ -38,5 +38,6 @@ server {
     ssl_certificate /usr/share/xivo-certs/server.crt;
     ssl_certificate_key /usr/share/xivo-certs/server.key;
     ssl_ciphers ALL:!aNULL:!eNULL:!LOW:!EXP:!RC4:!3DES:!SEED:+HIGH:+MEDIUM;
-    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_protocols TLSv1.2 TLSv1.3;
+    add_header Strict-Transport-Security "max-age=31536000";
```

Comment:
After digging some more, I think HSTS will make things harder in some setups: when HSTS is enabled, one cannot use `https://<ip_address>` anymore.

So HSTS can only be used once the server has a domain name, and since Wazo Platform does not enforce having a domain name, it can break some use cases where the server is only accessed via IP address.

As a consequence, I think we should not enforce HSTS by default, but maybe make it easy to enable... I don't know exactly what kind of mechanism could be used though...

Maybe we can extract the HSTS part to another PR, so that we can merge the rest?

Reply:
> So HSTS can only be used once the server has a domain name

As per my readings, HSTS headers will be considered by browsers only for domain-name-based sites, so will be ignored for IP-based sites.
I performed a few tests to confirm this, reaching a HTTPS (self-signed) site through its public IP, with HSTS headers enabled. No issue at all (Chrome, Firefox, Safari).

Reply:
Oh, that's good news :) Objection dismissed then! Thank you! We should be able to merge the PR next week, because right now we're preparing version 22.08 that will ship next monday.

Reply:
Perfect, many thanks for your review @sduthil 👍
And good luck for 22.08 :)